subgraph/schema.graphql

```diff
@@ -199,4 +195,16 @@ interface Event {
   transaction: Transaction!
   "msg.sender for the event"
   sender: Bytes!
+}
+
+type ERC20 @entity(immutable: true) {
```

Comment:
if someone was to deploy an ERC20 that changed these values, and then traded it, would that brick the sg due to this `immutable`?

Reply:
No, because the sg doesn’t update it. It only creates it the first time.